### PR TITLE
Small textual changes to SP Site Theming article

### PR DESCRIPTION
**Pull request by KoenZomers**
#### Category
- [X] Content fix
- [ ] New article

#### Related issues:
N/A

#### What's in this Pull Request?

Did some small textual changes to correct a statement on having to be a global admin to enable a site scoped feature and removed the quotes from the PnP PowerShell commands as these can be found confusing and copied along.

### CODE REVIEW

File: docs/declarative-customization/site-theming/sharepoint-site-theming-overview.md [hunk 1]
```diff
@@ -67,36 +67,34 @@ For the simplest experience, we recommend that you use modern themes with modern
 
 The modern site theming experience has been rolled out to classic site templates, too. While the new client-side theming architecture is more performant, if you have customizations on classic sites that aren’t rendering properly after you change the site’s theme, you can opt the site out of the new theming experience by disabling the feature. Please note, this opt-out only applies to classic sites where you have custom theme references that aren't rendering properly. By enabling this site-level opt-out you will disable the modern theming - and also lose the fast theming benefits it provides.
 
-To do this, you must use a Windows PowerShell script with a CSOM (client-side object model) wrapper. We recommend using the PnP enable feature command:
+To do this, you must use a Windows PowerShell script with a CSOM (client-side object model) wrapper. We recommend using the PnP PowerShell enable feature command:
```
Comment (andrewconnell) [inline docs/declarative-customization/site-theming/sharepoint-site-theming-overview.md]: Why did you remove "PowerShell"? There's no "PnP enable feature"...

Comment (KoenZomers) [reply docs/declarative-customization/site-theming/sharepoint-site-theming-overview.md]: I added PowerShell? It wasn't there before.

Comment (andrewconnell) [reply docs/declarative-customization/site-theming/sharepoint-site-theming-overview.md]: Oops... my bad. I had my eyes crossed... #darn-monday
